feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/data/repository/datasource/reward/Utils.kt

```diff
@@ -0,0 +1,25 @@
+package io.novafoundation.nova.feature_staking_impl.data.repository.datasource.reward
+
+import io.novafoundation.nova.common.data.network.subquery.SubQueryResponse
+import io.novafoundation.nova.common.utils.scopeAsync
+import io.novafoundation.nova.feature_staking_impl.data.network.subquery.response.StakingPeriodRewardsResponse
+import io.novafoundation.nova.feature_staking_impl.data.network.subquery.response.totalReward
+import io.novafoundation.nova.runtime.multiNetwork.chain.model.Chain
+import java.math.BigInteger
+import kotlinx.coroutines.awaitAll
+
+suspend inline fun getAggregatedRewards(
```

Comment:
This could be a protected member of `BaseStakingRewardsDataSource` - no need to make it top-level public function as this point